src/finder.js

```diff
@@ -32,7 +49,15 @@ export function isCommand (node) {
 }
 
 export function isNamespace (node) {
-  return type(node) === 'Object' && any(isCommand, values(node)) && node
+  return type(node) === 'Object' && any(v => isCommand(v) || isModule(v), values(node)) && node
+}
+
+export function isModule (node) {
+  return !isNil(node) && type(node.module) === 'String' && node
+}
+
+export function loadModule (node) {
+  return require(node.module).default
```

Comment:
I'd add compatibility for non-babel/ES6 modules (e.g `return require(node.module).default || require(node.module)`)